example/__tests__/App.js

```diff
@@ -79,31 +79,31 @@ describe('Async Storage mock functionality', () => {
   });
 
   describe('Callback based', () => {
-    it('can read/write data to/from storage', done => {
+    it('can read/write data to/from storage', (done) => {
```

Comment:
This seems like it's not our eslint config - could you check it out?

Reply:
Yeah, I bumped `@react-native-community/eslint-config` to ^2.0 which bumps Prettier to ^2.0. This is expected.

![image](https://user-images.githubusercontent.com/4123478/97080359-f3ba1d00-15fa-11eb-9f51-fd56cdcac6eb.png)